api/desecapi/dns.py

```diff
@@ -25,6 +25,15 @@ def _strip_quotes_decorator(func):
 dns.rdtypes.svcbbase.PortParam.to_text = _strip_quotes_decorator(dns.rdtypes.svcbbase.PortParam.to_text)
 
 
+@dns.immutable.immutable
+class CERT(dns.rdtypes.ANY.CERT.CERT):
```

Comment:
how is this different from base class? add short description

Reply:
upstream implementation calls _ctype_to_text. Added comment